### ISSUE TITLE
Geoengine view options

### ISSUE THREAD

**Issue by sciallo** [OCA/geospatial#92]
What options are possible in the following?

```
    <geoengine version="7.0">
        <field name="name" select="1"/>
        <field name="internal_id" select="1"/>
        <field name="type"/>
        <field name="height"/>
    </geoengine>
```

Specifically:
- What does `select="1"` do?
- How do I sort how fields are displayed? (I tried `sequence="nn" and order="nn", height field shows above type no matter what.
- I tried `<field name="height" invisible="1"/>` to make a field available to a colored representation, but the field is displayed 
- Is there any way to link to the form view of that particular record for more detail?
- What other elements/tags can be used inside the geoengine tag?

I apologize in advance if I should already know any of this, I have tried many things already and tried researching, I'm learning...

**Comment by yvaucher** [comment]
Hello @sciallo ,

The geoengine view is very simple and basic. Invisible as other formatting are not available.
`select` is an old attribute prior of the search view which was allowing to have the field in the search.
It is deprecated.
To filter what is displayed you must add it in the search view.

To use field for colourization it should be done on the vector layer object.
You have to set representation mode as coloured range, classification as coloured and choose on which field you want to do it with the field `attribute field`.

Here is an example in a XML definition:
https://github.com/OCA/geospatial/blob/9.0/base_geoengine_demo/retail_machine_view.xml#L116

On the part with linking a point to a form I'm working on it right now.

**Comment by ArchibaldJEGO** [comment]
Hello,

I'm interested in the link feature as well. Any idea when it will be available @yvaucher ?

Thanks

**Comment by yvaucher** [comment]
I made a [Pull Request](https://github.com/OCA/geospatial/pull/94)

You can test it here: http://3148002-94-ec85e4.runbot1.odoo-community.org/web/login with runbot.

**Comment by yvaucher** [comment]
Note that the development has been made only for 9.0 version. I have no plan to backport it in previous versions. (the call to switch view might be very different and I haven't the time to dig more)

**Comment by ArchibaldJEGO** [comment]
Thanks!

**Comment by yvaucher** [comment]
Note https://github.com/OCA/geospatial/pull/94 has been merged in 9.0
